### ISSUE TITLE
1.5.3 still breaks by not shipping qrcode/build/qrcode.js to npm

### ISSUE THREAD

**Issue by epoberezkin** [soldair/node-qrcode#333]

**Comment by ZhangChengLin** [comment]
https://www.npmjs.com/package/qrcode?activeTab=code

**Comment by epoberezkin** [comment]
indeed, and it was there in 1.5.1: https://www.npmjs.com/package/qrcode/v/1.5.1?activeTab=code

could it be published please?

**Comment by DanRyan483** [comment]
Agreed. It really should be included, especially since it is still being directly called out as an official use case for pre-compiled builds in the documentation: https://github.com/soldair/node-qrcode#precompiled-bundle

**Comment by btakita** [comment]
https://github.com/soldair/node-qrcode#browser

| node-qrcode can be used in browser through module bundlers like [Browserify](https://github.com/substack/node-browserify) and [Webpack](https://github.com/webpack/webpack) or by including the precompiled bundle present in build/ folder.

**Comment by shoyuf** [comment]
Please publish a new patch version to fix this problem

**Comment by eltoroit** [comment]
+1

**Comment by johnjago** [comment]
For anyone who finds this issue, two workarounds are mentioned in a newer issue: https://github.com/soldair/node-qrcode/issues/346

Install from GitHub: https://github.com/soldair/node-qrcode/issues/346#issue-1887598810

Install older version: https://github.com/soldair/node-qrcode/issues/346#issuecomment-1759642384
